### PR TITLE
Update to 0.9/0.10-alpha [Section 1]

### PR DESCRIPTION
**Pull request by savekirk**
This is the first PR towards closing #44. My goal is to update all the sections/pages of the book. After this PR is merged I'll submit different PRs for the rest of the pages.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,6 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - 1.39.0 # MSRV
```
Comment (JohnTitor) [inline .github/workflows/ci.yml:11]: This shouldn't be removed, specify 1.40.0 instead.

**Comment by savekirk** [comment]
Thanks for the review @JohnTitor. Currently the test are failing due to the other sections which are yet to be updated and fixing it will mean submitting a giant PR unless there's a way I can go around it. Thanks

**Comment by JohnTitor** [comment]
If you want to go step by step, I can prepare a branch like 0.9 on upstream and we could make progress there.

**Comment by savekirk** [comment]
@JohnTitor that works

**Comment by JohnTitor** [comment]
Pushed 0.9-next branch.

**Pull request closed without merging**